### ISSUE TITLE
I am trying to use multi retreival QA chain but not sure what version of langchain will be best?

### ISSUE THREAD

**Issue by Arham-12336** [hwchase17/langchain#7075]
### Issue you'd like to raise.

I am trying to develop a chatbot that can give answers from multiple namespace.That why I am try to use the MULTIRETREIVALQA chain but the langchain versions seems to be making some issues for me . Any suggestions?

**Comment by conboy** [comment]
Are you referring to vector databases like Pinecone? Also, please share the error messages you are getting so I can help out :+1:

**Comment by Arham-12336** [comment]
Yes I am referring pinecone database.There are issue with chatvectordbqa chain and importing of different modules

**Comment by conboy** [comment]
What modules are you trying to import?

**Comment by Arham-12336** [comment]
I am trying to use the fromLLMAndRetriever from the MultiRetrievalQAChain but cannot use it any help with that?

**Comment by jrcapicua** [comment]
I have the same problem...
Error:
Type 'VectorStoreRetriever<PineconeStore>' is missing the following properties from type 'VectorStoreRetriever<VectorStore>': lc_namespace, searchType, _vectorstoreType, _getRelevantDocuments, and 20 more

how to you finally fix this?

**Comment by Arham-12336** [comment]
> I have the same problem... Error: Type 'VectorStoreRetriever' is missing the following properties from type 'VectorStoreRetriever': lc_namespace, searchType, _vectorstoreType, _getRelevantDocuments, and 20 more
> 
> how to you finally fix this?

import { PineconeStore } from 'langchain/vectorstores/pinecone';

you can create a custom chain and add the following code to that
`vectorstore: pineconStore`

and  simply when returning the instance of the chain add this 
`const retriever=vectorstore.asRetriever();`
